website/docs/reference/resource-configs/materialized.md

```diff
@@ -1,7 +1,83 @@
-:::danger
+---
+resource_types: [models]
+description: "Materialized - Read this in-depth guide to learn about materializations in dbt."
+datatype: "string"
+---
 
-These docs are a placeholder for a yet-to-be-written reference section.
+<Tabs
+  groupId="config-languages"
+  defaultValue="project-yaml"
+  values={[
+    { label: 'Project file', value: 'project-yaml', },
+    { label: 'Property file', value: 'property-yaml', },
+    { label: 'Config block', value: 'config', },
+  ]
+}>
 
-Please refer to the [guide on materializations](/docs/build/materializations) for current documentation.
 
-:::
+<TabItem value="project-yaml">
+
+<File name='dbt_project.yml'>
+
+```yaml
+[config-version](/reference/project-configs/config-version): 2
+
+models:
+  [<resource-path>](/reference/resource-configs/resource-path):
+    +materialized: [<materialization_name>](https://docs.getdbt.com/docs/build/materializations#materializations)
+```
+
+</File>
+
+</TabItem>
+
+
+<TabItem value="property-yaml">
+
+<File name='models/properties.yml'>
+
+```yaml
+version: 2
+
+models:
+  - name: [<model-name>]
```

Comment:
Nit: the square brackets make it look like it is an array. And we're using an underscore rather than a dash elsewhere.

So I'd suggest the following:
```suggestion
  - name: <model_name>
```